specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-03-01/AutomationRules.json

```diff
@@ -942,17 +1220,47 @@
             "value": "Property",
             "description": "Evaluate an object property value"
           },
+          {
+            "value": "PropertyArray",
+            "description": "Evaluate an object array property value"
+          },
           {
             "value": "PropertyChanged",
             "description": "Evaluate an object property changed value"
           },
           {
             "value": "PropertyArrayChanged",
             "description": "Evaluate an object array property changed value"
+          },
+          {
+            "value": "Boolean",
+            "description": "Apply a boolean operator (e.g AND, OR) to conditions"
           }
         ]
       }
     },
+    "EntityManualTriggerRequestBody": {
+      "description": "Describes the request body for triggering a playbook on an entity.",
+      "required": [
+        "logicAppsResourceId"
+      ],
+      "type": "object",
+      "properties": {
+        "incidentArmId": {
```

Comment:
>Arm [](http://example.com/codeflow?start=17&length=3)

I'd name it 'incidentResourceId' since we try to avoid putting ARM in the name of things.

Reply:
You can also annotate it with `"format": "arm-id"`

Reply:
And perhaps [https://azure.github.io/autorest/extensions/#x-ms-arm-id-details](https://azure.github.io/autorest/extensions/#x-ms-arm-id-details)

Reply:
I added the "arm-id" annotation in the relevant places you specificed. I wasn't aware it existed :)
Regarding the field name - I agree the naming isn't ideal but I think it's better to leave as is since this feature has been released in a preview API a year ago (https://github.com/Azure/azure-rest-api-specs/blob/main/specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-03-01-preview/AutomationRules.json#L1253) with this naming so this is the field name customers are already familiar with and using. Changing it only in this new stable version would cause a discrepancy with the previous preview APIs, and changing in both would be a breaking change. wdyt?